_observing-your-data/ad/index.md

```diff
@@ -119,13 +119,41 @@ After you define the detector, the next step is to configure the model.
 
 ## Step 2: Configure the model
 
-1. Add features to your detector.
+Add features to your detector. A _feature_ is any field in your index that you want to analyze for anomalies. A detector can discover anomalies across one or more features. 
```

Comment:
@kaituo: Is this better?

```suggestion
Add features to your detector. A _feature_ is any field in your index that you want to analyze for anomalies, including aggregations or painless scripts. A detector can discover anomalies across one or more features. 
```

Reply:
Still can be improved. A feature is not field. A feature is an aggregation of a field or a painless script.